debian/rules

```diff
@@ -1,10 +1,11 @@
 #!/usr/bin/make -f
 
 include /usr/share/postgresql-common/pgxs_debian_control.mk
+CFLAGS := -fstack-protector-strong -D_FORTIFY_SOURCE=2 -O2 -z noexecstack -fpic -Wl,-z,relro -Wl,-z,now -Wformat -Wformat-security -Werror=format-security
```

Comment:
I think this should include any CFLAGS that are passed in by debian 
```suggestion
CFLAGS := $(CFLAGS) -fstack-protector-strong -D_FORTIFY_SOURCE=2 -O2 -z noexecstack -fpic -Wl,-z,relro -Wl,-z,now -Wformat -Wformat-security -Werror=format-security
```

Reply:
This makes sense. On my tests CFLAGS was usually empty but it does not mean that it is always be empty like that.

Thanks for the suggestion